WordPress/src/main/java/org/wordpress/android/ui/reader/discover/viewholders/ReaderPostViewHolder.kt

```diff
@@ -175,6 +183,7 @@ class ReaderPostViewHolder(
 
     private fun onMoreClicked(uiState: ReaderPostUiState, actions: List<SecondaryAction>, v: View) {
         // TODO malinjir the popup menu was reused from the legacy implementation. It needs to be refactored.
+        AnalyticsTracker.track(Stat.POST_CARD_MORE_TAPPED)
```

Comment:
@malinajirka - actually you are correct, it's need to be collected for all of reader. If we stick with ReaderPostViewHolder then it will cover all of reader instead of having to make sure it's in every other place that we listen for the event. thoughts?

Reply:
Yep I think that's correct.

Reply:
Removed & in commit 5c65bda